BetterConsoles.Colors/Extensions/FormatExtensions.cs

```diff
@@ -27,7 +28,19 @@ internal static FormatType Merge(this FontStyleExt style, ColorFormatType colorF
         public static uint BitCount(this FontStyleExt styles)
         {
 #if NETCOREAPP3_0_OR_GREATER || NET5_0_OR_GREATER
-            return Popcnt.PopCount((uint)styles);
+            if(RuntimeInformation.ProcessArchitecture == Architecture.X64 || RuntimeInformation.ProcessArchitecture == Architecture.X86) 
```

Comment:
Okay, lets try that again github...


```suggestion
            if(RuntimeInformation.ProcessArchitecture == Architecture.X64 || RuntimeInformation.ProcessArchitecture == Architecture.X86) 
            {
                return Popcnt.PopCount((uint)styles);
            }
```